draft/spec/index.html

```diff
@@ -672,14 +672,15 @@ <h2>Basic Structure</h2>
         <section id="manifest">
             <h2>Manifest</h2>
             <p>
-                The value of the <code>manifest</code> key is a JSON object, with keys corresponding to the digests
-                of every content file in the current and previous versions of the <a>OCFL Object</a>. The value for
-                each key <span id="E092">MUST</span> be an array containing the <a>content path</a>s of files in the
-                OCFL Object that have content with the given digest. As JSON keys are case sensitive, while digests may
-                not be, there is an additional requirement that each digest value <span id="E096">MUST</span> occur
-                only once in the manifest regardless of case. Content paths within a manifest block
-                <span id="E042">MUST</span> be relative to the <a>OCFL Object Root</a>. The following restrictions
-                avoid ambiguity and provide path safety for clients processing the <code>manifest</code>.
+                The value of the <code>manifest</code> key <span id="E106">MUST</span> be a JSON object, and each key
+                <span id="E107">MUST</span> correspond to a digest value key found in one or more <code>state</code>
+                JSON objects of the current and/or previous <code>version</code> blocks of the <a>OCFL Object</a>. The
```

Comment:
Do we call the state section a state JSON object in other locations?  I believe we generally refer to it as the state block.  I ask because the way this reads right now is a bit confusing and removing the `JSON objects` language might help that.

Reply:
Also noting that this may have impact on the codes section.

Reply:
You are right, we say "`state` block". Will fix

Reply:
We are inconsistent about use of code font for "`xyx` block" -- will use it here, suggest overall fix https://github.com/OCFL/spec/issues/562